### PR TITLE
Make `AddFunds` modals consistent accross pages

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/5658

⚠️ Require: https://github.com/opencollective/opencollective-api/pull/7769 and https://github.com/opencollective/opencollective-api/pull/8001

https://user-images.githubusercontent.com/12435965/179960380-04391072-1e62-43eb-9e4a-68f67d07e377.mp4

### CODE REVIEW

**Review by Betree — COMMENTED** [review]
I like the idea of fetching the permissions directly from the navbar. It is however a disruptive change compared to what we've done so far; we should try to have a consistent approach in how we do that in order to keep things simple for future developments. I've developed this proposal in https://github.com/opencollective/opencollective/issues/5970.

Regarding this PR, it could be a good MVP for the abovementioned issue. I need to give the review & testing some more attention.

File: components/collective-navbar/index.js [hunk 1]
```diff
@@ -183,6 +185,50 @@ const CategoriesContainer = styled(Container)`
...
+        }
+      }
+      ... on Fund {
+        permissions {
+          id
+          addFunds {
+            allowed
+            reason
+          }
+        }
```
Comment (Betree) [inline components/collective-navbar/index.js:226]: Why not directly:

```suggestion
      permissions {
        id
        addFunds {
          allowed
          reason
        }
      }
```

Comment (SudharakaP) [reply components/collective-navbar/index.js:226]: @Betree : This isn't possible I think as we don't have `permissions` on `Account`. In fact there's a ESlint warning if we do this; `ESLint: Cannot query field "permissions" on type "Account". Did you mean to use an inline fragment on "Bot", "Collective", "Event", "Fund", or "Host"?(graphql/template-strings)`

Comment (Betree) [reply components/collective-navbar/index.js:226]: The field was added to `AccountFields`, so the intent was clearly to add it to `Account`. Simply add the field definition in https://github.com/opencollective/opencollective-api/blob/d8551731c749832eb133928402709a95a5779846/server/graphql/v2/interface/Account.js#L61, it should solve the ESLint warning.

Comment (SudharakaP) [reply components/collective-navbar/index.js]: Thanks, I missed this. Done. 👍🏽

**Review by Betree — COMMENTED** [review]
No API PR? I don't understand how we ended up with `Account.permissions` set in the schema without one

**Comment by SudharakaP** [comment]
> No API PR? I don't understand how we ended up with `Account.permissions` set in the schema without one

Ops, sorry, I pushed the changes but somehow forgot to open the PR yesterday; just did that; https://github.com/opencollective/opencollective-api/pull/8001.

**Review by Betree — APPROVED** [review]
Looking good

**Pull request merged**